sklearn/feature_selection/_univariate_selection.py

```diff
@@ -236,10 +236,15 @@ def chi2(X, y):
 
     Notes
     -----
-    Complexity of this algorithm is O(n_classes * n_features).
+    - Complexity of this algorithm is O(n_classes * n_features).
+    - For continuous data, it is necessary to discretize the values (binning)
```

Comment:
Maybe add this after the "which must contain only **non-negative features** such as booleans or frequencies (e.g., term counts in document classification), relative to the classes." and only mention :class:`preprocessing.KBinsDiscretizer`

Reply:
Thanks a lot for your answer, I just did it.